Lesson22/sahra/accordion/styles.css

```diff
@@ -0,0 +1,72 @@
+@import url('https://fonts.googleapis.com/css2?family=Jaro:opsz@6..72&display=swap');
```

Comment:
It's better to add fonts and similar resources in HTML, CSS imports are usually less efficient.